Sources/iPhone/View Controllers/Main/DocumentsViewController_iPhone.m

```diff
@@ -82,20 +82,12 @@ -(void)viewDidLoad {
         [AppDelegate_iPhone instance].homeSidebarViewController_iPhone.contentNavigationItem.rightBarButtonItem = nil;
     }
 
-    if([eXoViewController isHighScreen]) {
-        CGRect tmpFrame = self.tblFiles.frame;
-        tmpFrame.size.height = iPHONE_5_SCREEN_HEIGH_MINUS_NAV_AND_STATUS_BAR;
```

Comment:
Check if the constant is used elsewhere or not. If not, delete it where it's defined.